src/main/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionsUpdateActionUtils.java

```diff
@@ -153,48 +149,51 @@
                     }
                 ));
 
-        return oldAttributeDefinitions
-            .stream()
-            .map(oldAttributeDefinition -> {
-                final String oldAttributeDefinitionName = oldAttributeDefinition.getName();
-                final AttributeDefinitionDraft matchingNewAttributeDefinitionDraft =
+        final List<UpdateAction<ProductType>> updateActions = new ArrayList<>();
+
+        for (AttributeDefinition oldAttributeDefinition : oldAttributeDefinitions) {
```

Comment:
why is it changed to a classical for-loop?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/316/commits/050df3b877dddd8f004ed05d403087a1ebad5754